### ISSUE TITLE
Make use of Jira's hour/day configuration when presenting time optional

### ISSUE THREAD

**Issue by damsgaard** [carstengehling/jirastopwatch#108]
When entering hours it is not logical to think of days for all people.

I therefore request that the feature introduced in 2.2.0 which used the server configuration for displaying and editing time, to be optional.

**Comment by carstengehling** [comment]
Good point. If you want it done fast, please submit a pull request. Have a look at this commit https://github.com/carstengehling/jirastopwatch/commit/d56c58748a2fde88ed2636e4ab35f4ec5d847785 where all the changes for the feature was done. This will give you an idea to what needs to be changed.

**Comment by damsgaard** [comment]
It pretty easy to make the change but I'm not a C# programmer and the UI part is not something I know about and this is required (checkmark) for it to be configurable.

I'll ask around.
